qiskit/circuit/instruction.py

```diff
@@ -426,13 +428,20 @@ def inverse(self):
         Special instructions inheriting from Instruction can
         implement their own inverse (e.g. T and Tdg, Barrier, etc.)
 
+        Args:
+            annotated: indicates whether the inverse gates can be implemented
+                as annotated gates.
+
         Returns:
             qiskit.circuit.Instruction: a fresh instruction for the inverse
```

Comment:
This isn't 100% true anymore as it's really `Instruction | AnnotatedOperation` now (as `AnnotatedOperation` doesn't inherit from `Instruction`).

Reply:
Good catch! I have improved the docstring for `Instruction.inverse()`.